ecs-cli/modules/cli/compose/container/container_test.go

```diff
@@ -126,6 +127,15 @@ func TestPortString(t *testing.T) {
 	}
 }
 
+func TestHealthStatus(t *testing.T) {
+	containerHealth := "SOME_VAL"
```

Comment:
Not sure if it's worth updating this to a realistic health value? eg. `ecs.HealthStatusHealthy`, `ecs.HealthStatusUnhealthy` or `ecs.HealthStatusUnknown`